From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mark D Horn <mark.d.horn@intel.com>
Date: Tue, 25 May 2021 13:29:47 -0700
Subject: [PATCH] Stateless patch

Signed-off-by: Mark D Horn <mark.d.horn@intel.com>
---
 agent/appconfig/constants_unix.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/appconfig/constants_unix.go b/agent/appconfig/constants_unix.go
index 842fc62..a48b39f 100644
--- a/agent/appconfig/constants_unix.go
+++ b/agent/appconfig/constants_unix.go
@@ -113,7 +113,7 @@ var (
 var PowerShellPluginCommandName string
 
 // DefaultProgramFolder is the default folder for SSM
-var DefaultProgramFolder = "/etc/amazon/ssm/"
+var DefaultProgramFolder = "/usr/share/defaults/etc/amazon/ssm/"
 
 var defaultWorkerPath = "/usr/bin/"
 var DefaultSSMAgentBinaryPath = defaultWorkerPath + "amazon-ssm-agent"
